pkg/revocation/revocation_db.go

```diff
@@ -1,44 +1,66 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-package identity
+package revocation
 
 import (
 	"context"
 	"crypto/x509"
 	"crypto/x509/pkix"
 
+	"github.com/zeebo/errs"
+	"gopkg.in/spacemonkeygo/monkit.v2"
+
 	"storj.io/storj/internal/dbutil"
+	"storj.io/storj/pkg/identity"
 	"storj.io/storj/pkg/peertls"
 	"storj.io/storj/pkg/peertls/extensions"
+	"storj.io/storj/pkg/peertls/tlsopts"
 	"storj.io/storj/storage"
 	"storj.io/storj/storage/boltdb"
 	"storj.io/storj/storage/redis"
 )
 
-// RevocationDB stores the most recently seen revocation for each nodeID
+var (
+	mon = monkit.Package()
+
+	// Error is a pkg/revocation error
+	Error = errs.Class("revocation error")
+)
+
+// DB stores the most recently seen revocation for each nodeID
 // (i.e. nodeID [CA certificate's public key hash] is the key, values is
 // the most recently seen revocation).
-type RevocationDB struct {
-	DB storage.KeyValueStore
+type DB struct {
+	KVStore storage.KeyValueStore
```

Comment:
why is this kvstore public?

Reply:
Because the old name `DB` was public, and it is referenced in the `testrevocation` package. This was simply a rename to avoid confusing duplicate `DB` variable names.